### ISSUE TITLE
Epic: WAVE: Alert: Link to PDF document

### ISSUE THREAD

**Issue by geolunalg** [hackforla/tdm-calculator#3310]
### Overview

We need to research, create, and document a solution for
Alert: Link to PDF document
Alert and apply it to the
affected page so the site can be WCAG compliant and so the solution is
available for future teams.

### Details
Category:  Alert 
Name: Link to PDF document
(The Known Issues Index has no solution yet for this issue)

Known page (there might be others):
1. My Projects

This epic will:
- Create a child issue to research, apply, and document a fix on the one
  known page
- After the fix is merged, verify whether the issue exists on any other pages

### Action Items
- [ ] Go to the Known Issues Solutions Index (resource 1.02) and add the link
      to the specific solutions page for this issue as resource 1.02.01
- [ ] Copy the child issue template (resource 1.04) and create a new issue
      for the `My Projects` page
   - [ ] Add the following labels to the new child issue:
      - `role: front-end`
      - `size: missing`
      - `p-feature: [REPLACE WITH PAGE NAME]`
      - `feature: Accessibility`
      - `level: missing`
      - `ready for dev lead`
      - `priority: MUST HAVE`
   - [ ] Add milestone: [REPLACE WITH MILESTONE NAME, if applicable]
   - Customize the child issue resources:
      - [ ] Update the child issue's resource 1.02.01 and 1.02.02 using
            the application details for this page
      - [ ] Find the spreadsheet row for this page and error, and update
            the child issue's resource 1.03 with the correct link and
            row number(s)
- [ ] Add the new issue as a sub-issue of this epic
- [ ] Once the child issue is merged and closed, run WAVE on all other pages
      for this [REPLACE WITH ONE OF: Alert / Contrast Error / Error]:
      [REPLACE WITH NAME OF Alert/Contrast Error/Error] only — ignore all
      other errors and alerts. Check user interaction states as well.
   - [ ] No additional instances found — close this epic
   - [ ] Additional instances found — create a new epic using the
         [Epic: Apply code solution for 1 or more pages](https://hackforla.github.io/accessibility/wiki-and-issue-templates/issue-template-epic-apply-solution/)
         template and link it here

### Resources
- 1.01 WAVE Accessibility Audit Spreadsheet, tab: Audit Log:
       https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=0#gid=0&fvid=582895304
   - 1.01.01 Audit Spreadsheet, filter:
             https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=0#gid=0&fvid=582895304
- 1.02 [WCAG Known Issues and Solutions](https://hackforla.github.io/accessibility/known-issues-and-solutions/)
   - 1.02.01 https://hackforla.github.io/accessibility/known-issues-and-solutions/solutions-index/[REPLACE WITH SLUG-WITH-DASHES]/
- 1.03 Project documentation / page inventory:
      https://hackforla.github.io/accessibility/wiki-and-issue-templates/issue-template-epic-create-apply-document-solution-1page/
   - 1.03.01 Documentation for My Projects page:

#### 1.04 Child Issue Template

Title

**Comment by geolunalg** [comment]
Technical Notes:

- The Alerts apear to be hidden, regarless of the logged in user. We need confirmation if this element (PDF) needs to be visible to the users.
- If the PDF is not supposed to be visible them the solution may be just to remove the element from the page conpleatly. This will also mean we will not be able to create a solution page.

element:
```
<a href="https://planning.lacity.org/odocument/bb9114b3-29e3-423f-8b91-027afb242e63/Revised_DRAFT_TDMOrdinance_June2022.pdf" target="external">
Draft Revised TDM Ordinance
<svg stroke="currentColor" fill="currentColor" stroke-width="0" viewBox="0 0 24 24" height="0" width="0" xmlns="http://www.w3.org/2000/svg">
<path fill="none" d="M0 0h24v24H0z"></path>
<path d="M19 19H5V5h7V3H5a2 2 0 0 0-2 2v14a2 2 0 0 0 2 2h14c1.1 0 2-.9 2-2v-7h-2v7zM14 3v2h3.59l-9.83 9.83 1.41 1.41L19 6.41V10h2V3h-7z"></path>
</svg>
</a>
```

<img width="1005" height="527" alt="Image" src="https://github.com/user-attachments/assets/50a0cd91-61fd-4494-8786-50fe3eb3ddda" />

<img width="1567" height="670" alt="Image" src="https://github.com/user-attachments/assets/678c0276-fde6-4257-b1e8-e9d32f259970" />

**Comment by geolunalg** [comment]
I have created the following two stories:
- https://github.com/hackforla/tdm-calculator/issues/3310
- https://github.com/hackforla/tdm-calculator/issues/3311

Need to review the last Allert in the filtered list: Underlined text

**Comment by geolunalg** [comment]
Closing this issue as it a issues as this items on the that are display by the WAVE tool are pdf file information.
